flink-table-store-core/src/main/java/org/apache/flink/table/store/file/utils/SnapshotManager.java

```diff
@@ -44,6 +45,8 @@ public class SnapshotManager {
     private static final String SNAPSHOT_PREFIX = "snapshot-";
     public static final String EARLIEST = "EARLIEST";
     public static final String LATEST = "LATEST";
+    private static final int READ_HINT_RETRY_NUM = 10;
+    private static final int READ_HINT_RETRY_INTERVAL = 200;
```

Comment:
Maybe the better choice is:
- `READ_HINT_RETRY_NUM` is 3.
- `READ_HINT_RETRY_INTERVAL` is 1ms.

Reply:
Done.